File: 02.DLL/WareHouse/Kho Moi/XtraForm1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraEditors;
using Microsoft.ApplicationBlocks.Data;

namespace WareHouse.Forms
{
    public partial class XtraForm1 : DevExpress.XtraEditors.XtraForm
    {
        public XtraForm1()
        {
            InitializeComponent();
        }

        private void XtraForm1_Load(object sender, EventArgs e)
        {
            DataTable dt = new DataTable();
            dt.Load(SqlHelper.ExecuteReader(Commons.IConnections.ConnectionString, "spInLichSuThietBi", "-1", "01/01/2000", "01/01/2022"));

            ucInLSTB myUc = new ucInLSTB(DateTime.Parse("01/01/2000"), DateTime.Parse("01/01/2002"), "-1","ten may", dt);
            myUc.Location = new Point(0, 0);
            myUc.Dock = DockStyle.Fill;
            this.Controls.Add(myUc);
            myUc.InReport();
            Commons.Modules.ObjSystems.ThayDoiNN(this);
        }

        
    }
}

File: 02.DLL/Report1/Report1/XtraForm1.vb
Public Class XtraForm1 
    Private sql As String
    Private data As New DataTable

    Private Sub XtraForm1_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        'Commons.Modules.SQLString = "0LOAD"


        Dim myUc As New ctlDMTBDL
        myUc.Location = New Point(0, 0)
        myUc.Dock = DockStyle.Fill
        Me.Controls.Add(myUc)
        Commons.Modules.ObjSystems.ThayDoiNN(Me)

    End Sub

    Private Sub cmbThang_EditValueChanged(ByVal sender As System.Object, ByVal e As System.EventArgs)
        If Commons.Modules.SQLString = "0LOAD" Then Exit Sub

    End Sub
End Class